SD-K9/src/app/components/building-info/building-info.component.ts

```diff
@@ -121,8 +121,8 @@ export class BuildingInfoComponent implements OnInit, OnDestroy {
     return true;
   }
 
-  toggleBuildingInfoCardTabs(e){
-    let toShow = e.detail.value;
+  toggleBuildingInfoCardTabs(event : Object){
```

Comment:
I think it's a MouseEvent, but you would need to try it out. If it still doesn't work I guess no choice, gotta use any. But at least developers will now know that this event is expected to be anything, so use at your own risk.

my source: [here](https://stackoverflow.com/questions/35966965/what-typescript-type-is-angular2-event)